### ISSUE TITLE
Lens-5.4.4 AppImage extract has started throwing error

### ISSUE THREAD

**Issue by gautam7730** [kontena/lens#6246]
**Describe the bug**
On the Ubuntu VM, when trying to extract the image for Lens-5.4.4, it gives below error:

```
sudo curl -LO https://api.k8slens.dev/binaries/Lens-5.4.4-latest.20220324.1.x86_64.AppImage

sudo chmod +x Lens-5.4.4-latest.20220324.1.x86_64.AppImage

ubuntu@lens-vm:~/test$ sudo ./Lens-5.4.4-latest.20220324.1.x86_64.AppImage --appimage-extract

sudo: unable to resolve host lens-vm: Name or service not known

./Lens-5.4.4-latest.20220324.1.x86_64.AppImage: 2: Syntax error: newline unexpected

```

**To Reproduce**
1. On Ubuntu vm, download the appImage using:
`sudo curl -LO https://api.k8slens.dev/binaries/Lens-5.4.4-latest.20220324.1.x86_64.AppImage
`
3. give permission:  `sudo chmod +x Lens-5.4.4-latest.20220324.1.x86_64.AppImage`
4. run the extract command: ` sudo ./Lens-5.4.4-latest.20220324.1.x86_64.AppImage --appimage-extract`
5. See error:
`
ubuntu@lens-vm:~/test$ sudo ./Lens-5.4.4-latest.20220324.1.x86_64.AppImage --appimage-extract
sudo: unable to resolve host lens-vm: Name or service not known
./Lens-5.4.4-latest.20220324.1.x86_64.AppImage: 2: Syntax error: newline unexpected
`

**Expected behavior**

**Screenshots**

**Environment (please complete the following information):**
- Lens Version: 5.4.4
- OS: Ubuntu
- Installation method (e.g. snap or AppImage in Linux): AppImage

**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
```
sudo curl -LO https://api.k8slens.dev/binaries/Lens-5.4.4-latest.20220324.1.x86_64.AppImage
sudo chmod +x Lens-5.4.4-latest.20220324.1.x86_64.AppImage
ubuntu@lens-vm:~/test$ sudo ./Lens-5.4.4-latest.20220324.1.x86_64.AppImage --appimage-extract
sudo: unable to resolve host lens-vm: Name or service not known
./Lens-5.4.4-latest.20220324.1.x86_64.AppImage: 2: Syntax error: newline unexpected
```

**Kubeconfig:**
Quite often the problems are caused by malformed kubeconfig which the application tries to load. Please share your kubeconfig, remember to remove any secret and sensitive information.
```
your kubeconfig here
```

**Additional context**

**Comment by gautam7730** [comment]
the below link for Lens 5.4.4 shows Access Denied:
https://downloads.k8slens.dev/ide/Lens-5.4.4-latest.20220324.1.x86_64.AppImage
If this link got changed? Can someone please share the updated link for this AppImage download?

**Comment by leenamba** [comment]
Hi all,
At Team Lens we take security very seriously. In order to keep Lens users safe and provide access to the latest features and fixes we have stopped distributing older versions of Lens. As well, to increase user safety we have introduced(since Lens 5.5) an update mechanism similar to the most popular web browsers making it easy to stay up to date with the latest version of Lens to ensure the most secure experience.
